airbyte-integrations/connectors/source-marketo/source_marketo/spec.json

```diff
@@ -0,0 +1,40 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/marketo",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Source Marketo Spec",
+    "type": "object",
+    "required": ["base_url", "client_id", "client_secret", "start_date"],
+    "additionalProperties": false,
+    "properties": {
+      "base_url": {
```

Comment:
We could name it as: `domain_url` for clarification.

Reply:
Updated.